database/factories/ModelFactory.php

```diff
@@ -21,3 +26,72 @@
         'visit_token' => $faker->asciify('********************'),
     ];
 });
+
+$factory->define(App\DuesPackage::class, function (Faker\Generator $faker) {
+    return [
+        'id' => $faker->numberBetween($min = 0, $max = 200),
+        'name' => $faker->word,
+        'eligible_for_shirt' => $faker->numberBetween($min = 0, $max = 1),
+        'eligible_for_polo' => $faker->numberBetween($min = 0, $max = 1),
+        'effective_start' => $faker->dateTime(),
+        'effective_end' => $faker->dateTime(),
```

Comment:
`effective_end` needs to be after `effective_start`. Maybe `dateTimeBetween()` will do that with some tweaking?